### PR TITLE
update NEOPIXEL_LED library

### PR DESCRIPTION
**Pull request by classicrocker883**
### Description
simple change to **features.ini**

```diff
-NEOPIXEL_LED                           = adafruit/Adafruit NeoPixel@~1.8.0
+NEOPIXEL_LED                           = adafruit/Adafruit NeoPixel@~1.12.3
```

### CODE REVIEW

**Comment by thisiskeithb** [comment]
Which MCU/families was this PR tested on?

I’d be careful bumping library versions without careful & thorough testing on real hardware.

**Review by classicrocker883 — COMMENTED** [review]
on STM32F103 using MKS_MINI_12864_V3

File: ini/features.ini [hunk 1]
```diff
@@ -28,7 +28,7 @@ HAS_MOTOR_CURRENT_(I2C|DAC|SPI|PWM)    = build_src_filter=+<src/gcode/feature/di
 HAS_MOTOR_CURRENT_I2C                  = SlowSoftI2CMaster
                                          build_src_filter=+<src/feature/digipot>
 LIB_INTERNAL_MAX31865                  = build_src_filter=+<src/libs/MAX31865.cpp>
-NEOPIXEL_LED                           = adafruit/Adafruit NeoPixel@~1.8.0
+NEOPIXEL_LED                           = adafruit/Adafruit_NeoPixel@~1.12.3
```
Comment (thisiskeithb) [inline ini/features.ini]: The failing CI test/build is obvious, but it should be `adafruit/Adafruit NeoPixel` (with a space).

See the Adafruit NeoPixel entry on the PlatfromIO Registry for more info: https://registry.platformio.org/libraries/adafruit/Adafruit%20NeoPixel/installation

Comment (classicrocker883) [reply ini/features.ini]: oh right, i was going by the github one whoops

**Pull request merged**